homeassistant/components/ecoal_boiler/sensor.py

```diff
@@ -47,3 +56,36 @@ def update(self) -> None:
         # Old values read 0.5 back can still be used
         status = self._ecoal_contr.get_cached_status()
         self._attr_native_value = getattr(status, self._status_attr)
+
+
+class EcoalFuelPercentageSensor(SensorEntity):
+    """Representation of a fuel level in percent."""
+
+    _attr_device_class = SensorDeviceClass.BATTERY
```

Comment:
This is not a battery sensor right?

Reply:
No it is not, but seemed like a good analogy. Couldn't find anything better in the documentation.